src/nft/components/collection/ActivityCells.tsx

```diff
@@ -269,25 +258,16 @@ export const EventCell = ({
   price,
   isMobile,
 }: EventCellProps) => {
-  const isNftGraphqlEnabled = useNftGraphqlEnabled()
-  const formattedPrice = useMemo(
-    () =>
-      price
-        ? isNftGraphqlEnabled
-          ? formatEth(parseFloat(price?.toString()))
-          : putCommas(formatEthPrice(price.toString()))?.toString()
-        : null,
-    [isNftGraphqlEnabled, price]
-  )
+  const formattedPrice = useMemo(() => (price ? formatEth(parseFloat(price?.toString())) : null), [price])
   return (
     <Column height="full" justifyContent="center" gap="4">
       <Row className={styles.eventDetail} color={eventColors(eventType)}>
         {renderEventIcon(eventType)}
         {ActivityEventTypeDisplay[eventType]}
       </Row>
-      {eventTimestamp && (isValidDate(eventTimestamp) || isNftGraphqlEnabled) && !isMobile && !eventOnly && (
+      {eventTimestamp && !isMobile && !eventOnly && (
```

Comment:
Why is it no longer being validated (`isValidDate`)?

Reply:
This has to do with the format passed back to us from the BE. From our now deprecated heroku api, date was passed as a string, non-unix timestamp, so this "validate" function made sure it could be parsed into unix. The gql endpoint returns dates now as straight unix so the validate function isn't needed